### PR TITLE
docs: Fix typos in README and CONTRIBUTING

### PR DESCRIPTION
**Pull request by pito-svk**
# PR o'clock

## Description

Fixed typos I've noticed while reading the docs.

### Checklist

- [x] CI tests are passing
- [x] README.md has been updated after any changes to variables and outputs. See https://github.com/terraform-aws-modules/terraform-aws-eks/#doc-generation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -168,7 +168,7 @@ MIT Licensed. See [LICENSE](https://github.com/terraform-aws-modules/terraform-a
 | Name | Description | Type | Default | Required |
 |------|-------------|------|---------|:--------:|
 | attach\_worker\_cni\_policy | Whether to attach the Amazon managed `AmazonEKS_CNI_Policy` IAM policy to the default worker IAM role. WARNING: If set `false` the permissions must be assigned to the `aws-node` DaemonSet pods via another method or nodes will not be able to join the cluster. | `bool` | `true` | no |
-| aws\_auth\_additional\_labels | Additionnal kubernetes labels applied on aws-auth ConfigMap | `map(string)` | `{}` | no |
+| aws\_auth\_additional\_labels | Additional kubernetes labels applied on aws-auth ConfigMap | `map(string)` | `{}` | no |
```
Comment (bryantbiggs) [inline README.md:171]: this should be fixed in the `variables.tf` and then it will be automatically fixed here when running `pre-commit run -a`

Comment (pito-svk) [reply README.md:171]: @bryantbiggs Thanks, I've updated variables.tf

**Review by bryantbiggs — APPROVED** [review]
@barryib 👍🏼

**Review by barryib — APPROVED** [review]
Thanks @pito-svk and @bryantbiggs

**Pull request merged**